locales/de/parameter.json

```diff
@@ -1,2 +1,23 @@
 {
+  "BOLUS_AGGRESSIVENESS_FACTOR": "Aggressivität bei Hyperglykämie",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏Some translation keys seem to be missing, for instance we used in our SYSTREQ-86 the value `IOB_TAU_S` and it's not in the file anymore

![image](https://github.com/mdblp/blip/assets/116577202/da3c65f4-c8ea-46ed-9ac4-7a17236c3d04)

Reply:
My advice would be to not try to remove any translation right now as it may break things, and instead review them later when we are not rushed by time

Reply:
Also, a lot of integration tests will need to be updated to use the translation value instead of the translation key: before your change, there were no translations for the parameters when the ITs were run so the ITs are based on the translation keys. (I can help you with that if needed)

Reply:
And finally, the test in `settings-print-view.util.test.ts` was also based on the translation keys instead of the translation values and it also needs to be updated to pass